.github/workflows/docker-images.yaml

```diff
@@ -0,0 +1,82 @@
+name: Build docker images
+on:
+  workflow_call:
+    inputs:
+      lang:
+        description: SDK language to build the container for (only Go is supported for now)
+        required: true
+        type: string
+      sdk-repo-ref:
+        description: Git ref of SDK repo to use to build (overrides "sdk-version")
+        required: false
+        type: string
+      sdk-repo-url:
+        description: URL of SDK repo to use to build (only used if "sdk-repo-ref" is provided)
+        required: false
+        type: string
+      sdk-version:
+        description: Version of SDK to use (ignored if "sdk-repo-ref" is provided)
+        required: false
+        type: string
+      semver-latest:
+        # TODO(bergundy): type: choice doesn't seem to work here
+        description: Tag images with major / minor versions (only used if "sdk-version" is provided - 'minor' or 'major')
+        type: string
+        default: 'none'
+      run-tag:
+        description: Docker tag suffix to use to test run the image (e.g. master or 1.16.0)
+        # TODO(bergundy): infer this from sdk-version / sdk-repo-ref
+        required: true
+        type: string
+
+jobs:
+  build-images:
+    runs-on: ubuntu-latest
+    env:
+      TEMPORAL_CLOUD_ADDRESS: sdk-ci.a2dd6.tmprl.cloud:7233
+      TEMPORAL_CLOUD_NAMESPACE: sdk-ci.a2dd6
+      REPO_URL: ${{ github.event.pull_request.head.repo.html_url }}
+    steps:
+      - name: Print build information
+        run: 'echo head_ref: ${{ github.head_ref }}, ref: ${{ github.ref }}'
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.17'
```

Comment:
Should bump up to at least 1.18 (1.17 is EOL, I need to fix in other places)

Reply:
I'll fix in one of the next PRs.